buddyboss/buddyboss-platform
# PROD-721 Fix - Few strings are not translatable

### All Submissions:


### Changes proposed in this Pull Request:
Fixes #3048 

### How to test the changes in this Pull Request:

1. Go to loco translate
2. BB platform plugin translation
3. Find the strings and translate those
4. See the strings on the frontend

### Proof Screenshots or Video
1. Code: https://prnt.sc/1vtbref
Front End: 
Got to Move document popup : https://ibb.co/d6WF61D
Click on "create new folder": https://ibb.co/sQjkssh

2. Code: https://prnt.sc/1vtbt0d
Front End: 
Cancel uploading file to see message: https://ibb.co/jgbW3qk

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
